examples/ccvm_boxqp_mf.py

```diff
@@ -15,12 +15,12 @@
     # Supply solver parameters for different problem sizes
     solver.parameter_key = {
         20: {
-            "pump": 0.5,
-            "feedback_scale": 20,
-            "j": 20,
-            "S": 0.2,
+            "pump": 0.0,
+            "feedback_scale": 4000,
+            "j": 5.0,
+            "S": 20.0,
             "dt": 0.0025,
-            "iterations": 15000,
+            "iterations": 1500,
```

Comment:
I assumed the change in values of some of the parameters is to reflect the change in the paper, right?

Reply:
Yes, that's correct! The large value of 15000 is no longer necessary as the solvers work find with only 1500 iterations.